xl2times/transforms.py

```diff
@@ -1266,44 +1206,93 @@ def _populate_calculated_defaults(df: DataFrame, model: TimesModel):
 
 def apply_fixups(
     config: Config,
-    tables: list[EmbeddedXlTable],
+    tables: dict[str, DataFrame],
     model: TimesModel,
-) -> list[EmbeddedXlTable]:
-    def apply_fixups_table(table: EmbeddedXlTable):
-        tags = [Tag.fi_t, Tag.uc_t]
-        if table.tag not in tags:
-            return table
-
-        df = table.dataframe.copy()
+) -> dict[str, DataFrame]:
+    """Apply fixups to the tables."""
+    exclude_tags = {Tag.tfm_mig, Tag.tfm_upd}
+    attribute_tags = [Tag.fi_t, Tag.uc_t]
+    currency = tables[Tag.currencies]["currency"][0]
+    mapping_to_defaults = {
+        "limtype": "limtype",
+        "timeslice": "tslvl",
+        "year2": "year2",
+    }
 
-        # TODO: should we have a global list of column name -> type?
+    for tag, dataframe in tables.items():
+        df = dataframe.copy()
```

Comment:
```suggestion
    for tag, dataframe in tables.items():
```
I don't think we need it here either. We never use `dataframe` again in this function.